integration_tests/e2e/health.cy.ts

```diff
@@ -4,6 +4,7 @@ context('Healthcheck', () => {
       cy.task('reset')
       cy.task('stubAuthPing')
       cy.task('stubIncentivesApiPing')
+      cy.task('stubOffenderSearchApiPing')
```

Comment:
i think you might need to add offender search to the health checks in this file actually 🤔 
https://github.com/ministryofjustice/hmpps-incentives-ui/blob/main/server/services/healthCheck.ts

[like in nonna](https://github.com/ministryofjustice/hmpps-non-associations/blob/main/server/services/healthCheck.ts#L61)

Reply:
Ah yes - so is stubbing it here not necessary? Why are we stubbing the others?

Reply:
we _do_ want to stub it once the health check actually exists so then health endpoint for offender search _would_ be called